docs/sources/mimir/configure/configure-zone-aware-replication.md

```diff
@@ -69,7 +69,7 @@ With a replication factor of 3, which is the default, deploy the Grafana Mimir c
 Deploying Grafana Mimir clusters to more zones than the configured replication factor does not have a negative impact.
 Deploying Grafana Mimir clusters to fewer zones than the configured replication factor can cause writes to the replica to be missed, or can cause writes to fail completely.
 
-If there are fewer than `floor(replication factor / 2)` zones with failing replicas, reads and writes can withstand zone failures.
+If there are fewer than `ceil(replication factor / 2)` zones with failing replicas, reads and writes can withstand zone failures.
```

Comment:
This entire phrasing is confusing and backwards to me. Instead of saying "fewer than X zones with failing replicas" can we say "There can be at most X zones with failing replicas otherwise reads and writes will fail"?

Reply:
Agree w/ your sentiment @56quarters.

Reply:
Rewrote it according to your suggestion, PTAL.